chain/chain-primitives/src/error.rs

```diff
@@ -70,9 +70,9 @@ pub struct Error {
 
 #[derive(Clone, Eq, PartialEq, Debug, Fail)]
 pub enum ErrorKind {
-    /// The block doesn't fit anywhere in our chain.
-    #[fail(display = "Block is unfit: {}", _0)]
```

Comment:
I think this unfit message is very confusing, it sounds like the block is invalid, so I renamed it.